### ISSUE TITLE
TOR phosphoproteome app and news item

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1801]
add  this to our external resources page
    https://www.pombase.org/community/internet-resources

Also add a news item
The TOR-dependent phosphoproteome is now available in PomBase (link
    https://www.pombase.org/reference/PMID:34296454)
Time-dependent changes can be viewed using the "Shinyapp' accompanying the publication 
    https://tmak.shinyapps.io/TM_App-Table_EV3/

@kimrutherford  it would be useful if we could add a link to the app to the publication page. Is this possible? It might be a useful feature to be able to link a publication page to an external app or dataset.

**Comment by kimrutherford** [comment]
>  it would be useful if we could add a link to the app to the publication page. Is this possible? It might be a useful feature to be able to link a publication page to an external app or dataset.

Yep, that's possible with the `reference_page_extra_links` configuration setting.  What text should the link have?
"Time-dependent changes can be viewed using the accompanying application"?

**Comment by ValWood** [comment]
I think that works. Once live I can ask GTiff if she wants. to suggest any tweaks.

**Comment by kimrutherford** [comment]
The link should be visible in the morning.

**Comment by ValWood** [comment]
All done!

**Comment by kimrutherford** [comment]
For Val: need to look at the text of the link, maybe improve it.

**Comment by ValWood** [comment]
You will also need to remind me where and how these links are configured, and how they relate to 

Configuring links
https://github.com/pombase/website/issues/1801
https://github.com/pombase/website/issues/1802
https://github.com/pombase/website/issues/629
https://github.com/pombase/website/issues/1821
https://github.com/pombase/website/issues/717

we should look at all of these,  decide a long term plan, and create documentation . I remember some of the history, but not all of it ;)
